dhall/tests/Dhall/Test/Dhall.hs

```diff
@@ -161,7 +161,12 @@ data NonEmptyUnion = N0 Bool | N1 Natural | N2 Text
 data Enum = E0 | E1 | E2
     deriving (Eq, Generic, Inject, Interpret, Show)
 
-data Records = R0 {} | R1 { a :: () } | R2 { x :: Double } | R3 { a :: (), b :: () } | R4 { x :: Double, y :: Double }
+data Records
+    = R0 {}
+    | R1 { a :: () }
+    | R2 { x :: Double }
+    | R3 { a :: (), b :: () }
+    | R4 { x :: Double, y :: Double }
```

Comment:
BTW I don't see any tests for simple record types like `data D = D { a :: Bool, … }` and newtypes – I assume they would be translated to records and also be influenced by the `collapseSingletonRecords` setting?!

Reply:
Did you see my comment above, @Gabriel439?

I'm mostly wondering whether these types are marshalled differently with the new default setting.